### ISSUE TITLE
PennDoT Bug: Server error::The target principal name is incorrect

### ISSUE THREAD

**Issue by ppator1** [ARA-Trans/iAM#517]
![image](https://app.gitkraken.com/api/glo/boards/5be1aa981fc3b8000f7d72ea/attachments/5dd5948f5b8585000f1fa40e)

**Comment by ppator1** [comment]
SP - 5

**Comment by ppator1** [comment]
Per Adam Clugh's email 11-21-2019:
"Good morning,

Thank you to everyone for all of the suggestions.  I worked with our SQL database team on this issue since a lot of the suggestions point to the connection.config file.  A maintenance was performed the morning of November 19th, which is the same day the issue started occurring in dev.  There seems to be an issue with the SSL certs after the maintenance.  We added a line to the connection string as shown below.  This has resolved the issue and the site is working again.  After all the application and SQL servers reboot during the Sunday morning IPL, we will remove this parameter and see if the issue still occurs.  I’ll send out an update on Monday with the status."
